bottles/frontend/main.py

```diff
@@ -91,6 +91,10 @@ def __init__(self):
         self.__create_action('import', self.__show_importer_view, ['<primary>i'])
         self.__create_action('preferences', self.__show_preferences, ['<primary>comma'])
         self.__create_action('help', self.__help, ['F1'])
+        self.__create_action('new', self.__new_bottle, ['<primary>n'])
+        self.__create_action('switch_to_bottles_page', self.__show_bottles_view, ['<alt>1'])
+        self.__create_action('switch_to_library_page', self.__show_library_view, ['<alt>2'])
+        self.__create_action('go_back', self.__go_back, ['<alt>Left'])
```

Comment:
Adding Escape would help.

```suggestion
        self.__create_action('go_back', self.__go_back, ['<alt>Left', 'Escape'])
```